enterprise_access/apps/api_client/constants.py

```diff
@@ -0,0 +1,16 @@
+"""
+Constants for API client
+"""
+import backoff
+from requests.exceptions import ConnectionError as RequestsConnectionError
+from requests.exceptions import HTTPError
+from requests.exceptions import Timeout as RequestsTimeoutError
+
+# wait_gen references the field in the backoff decorator representing
+# the backoff strategy for retries
+wait_gen = backoff.expo
+
+exception = (
```

Comment:
Should be plural. Perhaps backoff_exceptions or retry_exceptions?

Reply:
Updated 👍🏽